package.json

```diff
@@ -130,7 +130,7 @@
     "eslint-plugin-compat": "^4.0.2",
     "fetch-ponyfill": "^7.1.0",
     "ipfs": "^0.64.2",
-    "ipfs-geoip": "^8.0.0",
+    "ipfs-geoip": "file:../ipfs-geoip",
```

Comment:
temporary, this will be gone when https://github.com/ipfs-shipyard/ipfs-geoip/pull/93 and https://github.com/ipfs-shipyard/ipfs-geoip/pull/94 releases

Reply:
PS: You can target packages on github so we can see CI passing, similar to what was done at https://github.com/ipfs/interop/pull/499/commits/36aeeca4eaa2352a9725247fa236ccfc9268d318

Reply:
I found a really weird behaviour with `esbuild` when the dependency is a local project, build works just fine and I validated the minified js and it contains the `toLong` implementation which is needed for lookup.

However, when dependency is https://gitpkg.vercel.app/ipfs-shipyard/ipfs-geoip?refactor/dag-cbor it throws the `Could not resolve "os"` 🤔